www-src/app/replicator/change/change_dispatcher.coffee

```diff
@@ -37,8 +36,9 @@ module.exports = class ChangeDispatcher
     dispatch: (doc, callback = ->) ->
         log.info "dispatch #{doc.docType}"
 
-        if @isDispatched(doc) and @changeHandlers[doc.docType.toLowerCase()]
```

Comment:
No ! It's used here https://github.com/jacquarg/cozy-mobile/blob/development/www-src/app/replicator/replication_launcher.coffee#L62